vmm/src/lib.rs

```diff
@@ -293,6 +293,27 @@ impl Serialize for PciDeviceInfo {
     }
 }
 
+pub fn feature_list() -> Vec<String> {
+    vec![
+        #[cfg(feature = "dbus_api")]
```

Comment:
nit: Can you put a comment in the Cargo.toml file next to the features list pointing at this so that we don't forget we need to update it for new features.

Reply:
Good advice. Updated!